src/courses/mappers/06.md

```diff
@@ -0,0 +1,347 @@
+  }
+]
+```
+:::
+
+### 5. Additional Structures
+
+There are additional structures in the OHDF schema which are used for metadata/extraneous information storage. These exist alongside the <i>profiles</i> structure on the top level of the HDF schema. The general structure for the top level of the HDF schema is as follows:
+
+Field Name | Definition
+-|-
+`platform` | Information on the platform producing the export.
+`platform.name` | Name of platform export was run on.
+`platform.release` | Platform version.
+`platform.target_id` | Platform target ID (i.e., further identifying information on platform).
+`statistics` | Statistics relating to target scan run.
+`statistics.controls` | Break down of control statistics by result status.
+`statistics.duration` | Duration of run (in seconds).
+`profiles` | Profiles structure.
+`passthrough` | Extraneous information storage.
+`passthrough.auxiliary_data` | Storage for unused data from the sample file.
+`passthrough.auxiliary_data.name` | Name of auxiliary data source.
+`passthrough.auxiliary_data.data` | Auxiliary data.
+`passthrough.raw` | Raw data dump of input security service export.
+
+#### Knowledge Check
+
+Given a snippet of GoSec data, try and identify what information is applicable to the additional structures and which field they would correlate with.
+
+::: details GoSec Source Data
+```json
+{
+  "Golang errors": {},
+  "Issues": [
+    {
+      "severity": "MEDIUM",
+      "confidence": "HIGH",
+      "cwe": {
+        "id": "22",
+        "url": "https://cwe.mitre.org/data/definitions/22.html"
+      },
+      "rule_id": "G304",
+      "details": "Potential file inclusion via variable",
+      "file": "C:\\Users\\AGILLUM\\OneDrive - The MITRE Corporation\\Documents\\Code\\grype-0.34.4\\internal\\file\\tar.go",
+      "code": "82: \t\tcase tar.TypeReg:\n83: \t\t\tf, err := os.OpenFile(target, os.O_CREATE|os.O_RDWR, os.FileMode(header.Mode))\n84: \t\t\tif err != nil {\n",
+      "line": "83",
+      "column": "14",
+      "nosec": false,
+      "suppressions": null
+    }
+  ],
+  "Stats": {
+    "files": 199,
+    "lines": 12401,
+    "nosec": 0,
+    "found": 7
+  },
+  "GosecVersion": "dev"
+}
+```
+:::
+
+::: details OHDF Converted GoSec Data
+```json
+{
+  "platform": { "name": "Heimdall Tools", "release": "2.10.8" },
+  "version": "2.10.8",
+  "statistics": {},
+  "profiles": [],
+  "passthrough": {
+    "auxiliary_data": [{ "name": "Gosec", "data": { "Golang errors": {} } }]
+  }
+}
+```
+:::
+
+## Knowledge Check
+
+:::details What is the purpose of <i>Profiles</i>, <i>Controls</i>, and <i>Results</i>?
+- <i>Profiles</i> provides a high-level overview of the security tool result and the targeted system (i.e., metadata).
+
+- <i>Controls</i> provides information on the security requirements being applied to the target system.
+
+- <i>Results</i> provides information on the actual checks to ensure that the target system is compliant with the security requirements.
```

Comment:
change "is compliant with"
to "is meeting"

Reply:
Addressed.